### ISSUE TITLE
5.4.0 beta .1 crashing

### ISSUE THREAD

**Issue by Tux0Topo** [Ultimaker/Cura#15765]
### Application Version

5.4.0 beta .1

### Operating System

Windows 11

### Printer

Ender 3 v2 + Hellbot Magna 2 Dual 300

### Reproduction steps

Just try to start the app.

### Actual results

Error traceback:
[teacerback.log](https://github.com/Ultimaker/Cura/files/11772400/teacerback.log)

Logs:
[cura.log](https://github.com/Ultimaker/Cura/files/11772331/cura.log)
[stderr.log](https://github.com/Ultimaker/Cura/files/11772393/stderr.log)
[stdout.log](https://github.com/Ultimaker/Cura/files/11772394/stdout.log)

### Expected results

Just to start the app

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Additional information & file uploads

Deleted the entire configuration folder, but didn´t change anything.
The 5.4.0 alpha worked fine.
The 5.3.1 stable, is working fine.

The PC is running an i7 11800hq + RTX 3080.

**Comment by MariMakes** [comment]
Hey @Tux0Topo,

Thanks for the report 💪 
That doesn't look good 😨 

This looks like the type of crash we reported internally. It would show up if you would load one 5.4 to another 5.4 version.  
It was a cash issue related to: [the implementation of this feature.](https://github.com/Ultimaker/Cura/pull/15639). I was under the impression that it was resolved for everyone, but seeing this report, I'm not sure if that's the case.
I'll have to double check with the team. 

You might need to do a clean install. 🤔
This is how you perform a clean install
	1. Quit the application
	2. In a file explorer window, navigate to:
			My Computer/users/[user_name]/AppData/Local/cura
			My computer/Users//AppData/Roaming/cura
	1. Delete the entire 'cura' folder of your current version. Note: Cura folders of previous versions will be imported if not deleted!
	2. Open Ultimaker Cura. If it has been reset correctly, you will be greeted with the onboarding flow
	
If you do this you will use any custom profiles or printers. If you want to keep these make sure to make a backup.
You can read more about them here: https://support.makerbot.com/s/article/1667412704080

I hope this helps you get unstuck

**Comment by Tux0Topo** [comment]
Thanks!!! That worked just fine! Cant wait to print the dragon penholder HAHAHAHAHAHA

**Comment by casperlamboo** [comment]
@Tux0Topo thanks for you bug report, we are trying to resolve the issue but unfortunately we have somewhat of a hard time reproducing the error.

Did you by any change install any other 5.4-alpha version previously? for example the nightly build we shared to test the qt upgrade https://github.com/Ultimaker/Cura/discussions/15098.

Could you try removing both the roaming and local cura configurations folders and see if that resolves the issue?
```
c:\Users\<your username>\AppData\Roaming\cura\
c:\Users\<your username>\AppData\Local\cura\
```

**Comment by MariMakes** [comment]
Quick update from our side 👋

It's resolved in the 5.4 stable release, you can download the Cura version with the fix here: https://github.com/Ultimaker/Cura/releases/tag/5.4.0
I'll close this issue since it's resolved. 

Thanks again, and please let us know if you run into any other issues 💪
